### PR TITLE
Automate backup-utils releases

### PR DESCRIPTION
**Pull request by lildude**
This is a continuation of https://github.com/github/backup-utils/pull/299 which was accidentally closed when I accidentally committed the changes in that PR to master during the release process.

/cc @github/backup-utils

### CODE REVIEW

File: script/release [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+require 'net/http'
+require 'time'
+require 'erb'
+require 'English'
+
+API_HOST = ENV['GH_HOST'] || 'api.github.com'
+API_PORT = 443
+GH_REPO = ENV['GH_REPO'] || 'backup-utils'
+GH_OWNER = ENV['GH_OWNER'] || 'github'
+GH_AUTHOR = ENV['GH_AUTHOR'] || 'Sergio Rubio <rubiojr@github.com>'
```
Comment (rubiojr) [inline script/release]: I'd make this env var mandatory, I don't want to be credited for the work others did 😜.

File: RELEASING.md [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
+## Automatic Process from chatops (internal to GitHub only)
+
+1. `.ghe backup-utils-release 2.12.0`
+
+## Automatic Process from CLI
+
+1. Install the Debian `devscripts` package:
+  `sudo apt-get install devscripts`
+2. Run `script/release 2.12.0`
+
```
Comment (rubiojr) [inline RELEASING.md]: We should mention that a token and a number of env vars are required.

**Comment by lildude** [comment]
@rubiojr Done.

**Comment by lildude** [comment]
Ooops. This was once against accidentally merged. But on the plus side, it worked a treat for the 2.11.1 release

**Comment by lildude** [comment]
Note to self, don't test the automation from the branch that belongs to a PR for the automation :blush:

@rubiojr Want me to revert and try again or are you OK with things as they are in this PR which accidentally got merged?

**Comment by rubiojr** [comment]
@lildude works for me.

**Pull request merged**